packages/brokeneck-html/public/index.html

```diff
@@ -13,6 +13,8 @@
       rel="stylesheet"
       href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
     />
```

Comment:
do we still need roboto? it's needed by default with MUI, but if we're overriding it at all times then it may not be needed anymore

Reply:
I believe this is indeed completely overridden by the `typography` section in the updated theme. I've removed it from brokeneck-html and the custom path example.